src/de/hhn/aib/labsw/blackmirror/controller/API/SendPackage.java

```diff
@@ -0,0 +1,9 @@
+package de.hhn.aib.labsw.blackmirror.controller.API;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+public class SendPackage {
```

Comment:
does this need to be a mutable?
have a look at jdk14's record: https://www.baeldung.com/java-record-keyword